nvm.sh

```diff
@@ -1853,15 +1853,23 @@ nvm_install_source() {
 
   local make
   make='make'
-  if [ "${NVM_OS}" = 'freebsd' ]; then
-    make='gmake'
-    MAKE_CXX='CXX=c++'
-  elif [ "${NVM_OS}" = 'aix' ]; then
-    make='gmake'
-  fi
-  if nvm_has "clang++" && nvm_has "clang" && nvm_version_greater_than_or_equal_to nvm_clang_version 3.5 ; then
-    nvm_echo "Clang v3.5+ detected! Use Clang as c/c++ compiler!"
-    MAKE_CXX='CC=clang CXX=clang++'
+  case "${NVM_OS}" in
+    'freebsd')
+      make='gmake'
+      MAKE_CXX="CC=${CC:-cc} CXX=${CXX:-c++}"
```

Comment:
hm, this seems like an oversight - there's no `local MAKE_CXX` declaration in this function. let's add one above the `case`.